src/promptflow/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## v1.11.0 (Upcoming)
 
 ### Features Added
+
+- Introducing flex flow - design powerful LLM apps with the flexibility of Python functions or classes, and seamlessly test and run your logic with our VS Code Extension. Learn more about flex flow [here](https://microsoft.github.io/promptflow/how-to-guides/develop-a-flex-flow/index.html)
```

Comment:
we should add prefix `[promptflow-devkit]` and also add same message in `CHANGELOG.md` of `promptflow-devkit` package?

Reply:
Offline synced with @wangchao1230 , let's move them to announcement since they're not a devkit only feature.